packages/stats/tests/test_gbstats.py

```diff
@@ -226,230 +260,19 @@
 )
 
 
-class TestDiffDailyTS(TestCase):
```

Comment:
Why are we deleting all of these tests?

Reply:
i removed tests on a dummy branch (i thought) to make scrolling back and forth in the file faster, all tests are now restored.